addons/crm_iap_mine/models/crm_iap_lead_mining_request.py

```diff
@@ -333,6 +334,7 @@ def action_get_lead_action(self):
         self.ensure_one()
         action = self.env["ir.actions.actions"]._for_xml_id("crm.crm_lead_all_leads")
         action['domain'] = [('id', 'in', self.lead_ids.ids), ('type', '=', 'lead')]
+        action['context'] = dict(ast.literal_eval(action.get('context', '{}')), create=False)
```

Comment:
(Let's do it for both actions for consistency, sorry if that was not clear)

Reply:
Done, I removed the reformatting of contexts in xml too, since it was only necessary for literal eval to work properly